res/res_usbradio.c

```diff
@@ -319,15 +357,15 @@ int ast_radio_hid_device_mklist(void)
 	usb_find_devices();
 	for (usb_bus = usb_busses; usb_bus; usb_bus = usb_bus->next) {
 		for (dev = usb_bus->devices; dev; dev = dev->next) {
-			if ((dev->descriptor.idVendor
-				 == C108_VENDOR_ID) &&
+			if (((dev->descriptor.idVendor == C108_VENDOR_ID) &&
 				(((dev->descriptor.idProduct & 0xfffc) == C108_PRODUCT_ID) ||
 				 (dev->descriptor.idProduct == C108B_PRODUCT_ID) ||
 				 (dev->descriptor.idProduct == C108AH_PRODUCT_ID) ||
 				 (dev->descriptor.idProduct == C119A_PRODUCT_ID) ||
 				 (dev->descriptor.idProduct == C119B_PRODUCT_ID) ||
 				 ((dev->descriptor.idProduct & 0xff00) == N1KDO_PRODUCT_ID) ||
-				 (dev->descriptor.idProduct == C119_PRODUCT_ID))) {
+				 (dev->descriptor.idProduct == C119_PRODUCT_ID))) ||
+				 is_user_device(dev)) {

```

Comment:
Here, the indentation feels a bit off.  I think lines 361-367 need to be indented by one space and line 368 needs to be pulled back by one space.  But, maybe to make the comparisons clear and minimize the need to see the matching parens, I think we should introduce a new is_supported_device() that would incorporate our known/supported/blessed vid/pid's.